jekyll/_cci2/migrating-from-travis.md

```diff
@@ -8,56 +8,47 @@ description: "Migrating from Travis CI"
-| os:               | [docker, machine, macos](https://circleci.com/docs/2.0/executor-types/)            | Rather than an os, we offer Docker, Linux VM, and MacOS execution environments.                      |
-| cache components: | [restore_cache:,](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) [save_cache:](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) | Use the restore and save cache features to control caching in the builds |
-| before_cache      | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | If you want to run any commands before you cache, simply place a run: step before your cache step(s) |
-| before_install:   | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | CircleCI doesn't separate commands into stages or types. Use run: steps to specify any arbitrary commands and order them per your needs. See [documentation](https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute) for usage of conditional steps |
-| install:          | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | " (see above)                                                                                        |
-| before_script     | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | " (see above)                                                                                        |
-| script:           | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | " (see above)                                                                                        |
-| after_script:     | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                 | " (see above)                                                                                        |
-| deploy:           | [deploy:](https://circleci.com/docs/2.0/configuration-reference/#deploy)           | Use the deploy: step to deploy build artifacts                                                       |
+| language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language. Instead, CircleCI allows you to use any of the major operating systems or [Docker images]({{ site.baseurl }}/2.0/circleci-images/#next-gen-language-images) as executors. Specify a Docker image with your needed prerequisites or use a VM and install other needed libraries (see pre-installed software for [Windows]({{ site.baseurl }}/2.0/hello-world-windows/), [macOS]({{ site.baseurl }}/2.0/testing-ios/#supported-xcode-versions), and [Linux]({{ site.baseurl }}/2.0/docker-to-machine/#pre-installed-software) images). Then, use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |
```

Comment:
```suggestion
| language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language; instead, choose an executor and use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |
```

I think this line should be simplified. The information about pre-installed software and the links to other documents should all be found in the already linked to document on executor-types (which can be done in another PR). We should aim to keep table elements from getting too text-heavy (both for contributors to the docs and readers).